utils/mail.py

```diff
@@ -102,15 +104,20 @@ def send_mail(subject, email_body, user_to=None, email_to=None, email_from=None,
     if settings.ALLOWED_EMAILS:  # for testing purposes, so we don't accidentally send emails to users
         email_to = [(username, email) for username, email in email_to if email in settings.ALLOWED_EMAILS]
 
+    full_subject = u'{0} {1}'.format(settings.EMAIL_SUBJECT_PREFIX, subject)
+    if extra_subject:
+        full_subject += u' - {0}'.format(extra_subject)
```

Comment:
this feels a bit messy to me, having both `+=` and the format. Maybe it'd be a bit easier to understand to just say `full_subject = "{}- {}".format(full_subject, extra_subject)`

Reply:
Addressed in 9f6f6a65292e534cb40f0854e28449691a4a5851